.travis.yml

```diff
@@ -36,18 +36,18 @@ matrix:
     - python: '3.5'
       name: "3.5 Xenial PYTHONOPTIMIZE=2"
       env: PYTHONOPTIMIZE=2
-    - env: DOCKER="alpine" DOCKER_TAG="master"
-    - env: DOCKER="arch" DOCKER_TAG="master" # contains PyQt5
-    - env: DOCKER="ubuntu-16.04-xenial-amd64" DOCKER_TAG="master"
-    - env: DOCKER="ubuntu-18.04-bionic-amd64" DOCKER_TAG="master"
-    - env: DOCKER="debian-9-stretch-x86" DOCKER_TAG="master"
-    - env: DOCKER="debian-10-buster-x86" DOCKER_TAG="master"
-    - env: DOCKER="centos-6-amd64" DOCKER_TAG="master"
-    - env: DOCKER="centos-7-amd64" DOCKER_TAG="master"
-    - env: DOCKER="amazon-1-amd64" DOCKER_TAG="master"
-    - env: DOCKER="amazon-2-amd64" DOCKER_TAG="master"
-    - env: DOCKER="fedora-29-amd64" DOCKER_TAG="master"
-    - env: DOCKER="fedora-30-amd64" DOCKER_TAG="master"
+    - env: DOCKER="alpine" DOCKER_TAG="6.2.x"
+    - env: DOCKER="arch" DOCKER_TAG="6.2.x" # contains PyQt5
+    - env: DOCKER="ubuntu-16.04-xenial-amd64" DOCKER_TAG="6.2.x"
+    - env: DOCKER="ubuntu-18.04-bionic-amd64" DOCKER_TAG="6.2.x"
+    - env: DOCKER="debian-9-stretch-x86" DOCKER_TAG="6.2.x"
+    - env: DOCKER="debian-10-buster-x86" DOCKER_TAG="6.2.x"
+    - env: DOCKER="centos-6-amd64" DOCKER_TAG="6.2.x"
+    - env: DOCKER="centos-7-amd64" DOCKER_TAG="6.2.x"
+    - env: DOCKER="amazon-1-amd64" DOCKER_TAG="6.2.x"
+    - env: DOCKER="amazon-2-amd64" DOCKER_TAG="6.2.x"
+    - env: DOCKER="fedora-29-amd64" DOCKER_TAG="6.2.x"
```

Comment:
Let's remove EOL Fedora 29, especially as it's failing.

https://en.wikipedia.org/wiki/Fedora_(operating_system)#Releases

```suggestion
```